packages/patternfly-4/react-core/src/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from './components';
 export * from './layouts';
+export * from './internal';
```

Comment:
kind of semantics, but if it is exported it is not really internals anymore. Maybe rename to `helpers` or something similar?

Reply:
i was thinking the same thing this morning actually... i'm up for this if others are.

Reply:
helpers sounds good

Reply:
ok, works for me. Will update soon.